pubpub/pubpub
# Use default browser bullet styles

This bullet change was initially made to fix some issues with how the suggestions were rendered for bulleted lists. That  interface has been refactored a few times since then such that styling the bullets is less important, and not at all worth dealing with this kind of bug for.

## Issue(s) Resolved
Fixes #2737 

## Test Plan
- Visit [the review instance ](https://pubpub-pipel-kalilsn-fi-nyyett.herokuapp.com) and set your community to `hdsr`
- Visit [the podcast page](https://pubpub-pipel-kalilsn-fi-nyyett.herokuapp.com/podcast) and observe floated images and bullets work fine now, and that the bullets are reasonably sized

## Screenshots (if applicable)

### [Before:](https://hdsr.mitpress.mit.edu/podcast)
<img width="1281" alt="image" src="https://github.com/pubpub/pubpub/assets/473542/28328a90-7f13-4dbc-8a39-a79b99d0009e">

### [After:](https://pubpub-pipel-kalilsn-fi-nyyett.herokuapp.com/podcast)
<img width="1253" alt="image" src="https://github.com/pubpub/pubpub/assets/473542/033c709b-12ca-4c7b-84a8-cb5dbabf1fc3">
